pkg/solana/logpoller/parser.go

```diff
@@ -0,0 +1,225 @@
+package logpoller
```

Comment:
This filename is going to have a merge conflict with @dhaidashenko 's PR which adds some of `logpoller/parser.go` from evm logpoller in the same location. Since this file already means something else in evm, I'd prefer to keep it that way in Solana and rename this to something else. (I think you mentioned you were renaming Parser to LogDataParser, maybe this can be `log_data_parser.go` ?

Reply:
Ah, is `ProgramEventProcessor` what used to be `Parser`? If so, maybe this should be `event_processor.go`? Actually I'd be fine with `event_parser.go` too... maybe that makes more sense not sure.

Reply:
I can do that.